agent/config/config.go

```diff
@@ -93,6 +93,14 @@ const (
 
 	// pauseContainerTarball is the path to the pause container tarball
 	pauseContainerTarballPath = "/images/amazon-ecs-pause.tar"
+
+	// DefaultRpsLimit is set as 40. This is arrived from our benchmarking
+	// results where task endpoint can handle 4000 rps effectively. Here, 100 containers
+	// will be able to send out 40 rps.
+	DefaultRpsLimit = 40
+
+	// DefaultRpsBurstLimit is set to handle 60 burst requests at once
+	DefaultRpsBurstLimit = 60
```

Comment:
Similar to above. Can you please rename this to something like `DefaultTaskMetadataBurstRate`?